src/steps/campaign-member/campaign-member-id-equals.ts

```diff
@@ -18,16 +18,33 @@ export class CampaignMemberCampaignIdEquals extends BaseStep implements StepInte
     type: FieldDefinition.Type.STRING,
     description: 'Campaign ID',
   }];
+  protected expectedRecords: ExpectedRecord[] = [{
+    id: 'campaignMember',
+    type: RecordDefinition.Type.KEYVALUE,
+    fields: [{
+      field: 'Id',
+      type: FieldDefinition.Type.STRING,
+      description: "Campaign Member's SalesForce ID",
+    }, {
+      field: 'CreatedDate',
+      type: FieldDefinition.Type.DATETIME,
+      description: "Lead's Created Date",
+    }, {
+      field: 'LastModifiedDate',
+      type: FieldDefinition.Type.DATETIME,
+      description: "Lead's Last Modified Date",
```

Comment:
- [ ] Same, re: adding `CampaignId`, `LeadId`, `ContactId`, and `LeadOrContactId`, since these are all standard to the `CampaignMember` object and they would all be useful as tokens.